#!/bin/sh /usr/share/dpatch/dpatch-run
## 02-growisofs-manpage.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Fix manpage section

@DPATCH@

diff -Naurp dvd+rw-tools-7.1.orig/growisofs.1 dvd+rw-tools-7.1/growisofs.1
--- dvd+rw-tools-7.1.orig/growisofs.1   2008-03-01 10:40:06.000000000 +0000
+++ dvd+rw-tools-7.1/growisofs.1        2008-04-01 09:02:23.000000000 +0000
@@ -1,4 +1,4 @@
-.TH GROWISOFS 1m "1 Mar 2008" "growisofs 7.1"
+.TH GROWISOFS 1 "1 Mar 2008" "growisofs 7.1"
 .SH NAME
 growisofs \- combined mkisofs frontend/DVD recording program.
 .SH SYNOPSIS
